expr/api_test.go

```diff
@@ -132,29 +132,32 @@ func TestApiExpr_Hash(t *testing.T) {
 
 func TestDocsExprEvalName(t *testing.T) {
 	cases := map[string]struct {
-		url      string
-		expected string
+		url         string
+		expected    string
+		description string
 	}{
-		"test": {url: "http://parrot.com", expected: "Documentation http://parrot.com"},
+		"test": {url: "http://parrot.com", description: "A website for a parrot API", expected: "Documentation http://parrot.com"},
```

Comment:
Instead of replacing the existing test you could leave it and add new test cases:
```go
"uses name": {url: "http://parrot.com", expected: "Documentation http://parrot.com"},
"does not use description": {url: "http://parrot.com", description: "A website for a parrot API", expected: "Documentation http://parrot.com"},
"does not use documentation" ....
```

Reply:
Ah, I'm silly, sorry about that! That's been addressed in the latest commit. Thank you so much! :D